swan-lake/development-tutorials/source-code-dependencies/style-guide/annotations-documentation-and-comments.md

```diff
@@ -41,56 +41,32 @@ function testFunction() {
 **Example,**
   
 ```ballerina
+// Declares an annotation tag on the type.
+annotation record {||} validated on parameter, function, return;
+
 // Parameter annotation.
-public function secureFunction1(@untainted string secureInName, @untainted int secureInId, string insecureIn) {
-    ...
+public function secureFunction1(@validated string secureInName, int secureInId, string insecureIn) {
+...
 }
-  
-public function secureFunction2(@untainted string secureInName,
-    @untainted int secureInId, string insecureIn) {
-    ...
+
+public function secureFunction2(string secureInName,
+        int secureInId, string insecureIn) {
+...
 }
-  
+
 // Return type annotation.
-public function taintedReturn1() returns @tainted string {
-    ...
+@validated
```

Comment:
The comment doesn't say about func annotation. Hence let's remove this
```suggestion
```

Reply:
Updated.